CMakeLists.txt

```diff
@@ -8,6 +8,16 @@ project(KinesisVideoProducerCpp VERSION 3.4.0)
 set(CMAKE_CXX_STANDARD 11)
 include(GNUInstallDirs)
 
+set(KINESIS_VIDEO_PRODUCER_CPP_MAJOR_VERSION 3)
+set(KINESIS_VIDEO_PRODUCER_CPP_MINOR_VERSION 1)
+set(KINESIS_VIDEO_PRODUCER_CPP_PATCH_VERSION 1)
+set(KINESIS_VIDEO_PRODUCER_CPP_VERSION ${KINESIS_VIDEO_PRODUCER_CPP_MAJOR_VERSION}.${KINESIS_VIDEO_PRODUCER_CPP_MINOR_VERSION}.${KINESIS_VIDEO_PRODUCER_CPP_PATCH_VERSION})
+
+set(KINESIS_VIDEO_PRODUCER_CPP_MAJOR_VERSION 3)
+set(KINESIS_VIDEO_PRODUCER_CPP_MINOR_VERSION 1)
```

Comment:
Same here.

Reply:
What's the reason to have this twice?

Reply:
In the original, this was only there once. So I will remove the extra one.